packages/terra-application-layout/src/terra-dev-site/doc/example/ExampleApplication.jsx

```diff
@@ -330,10 +330,12 @@ ExampleApplication.propTypes = {
 const WrappedApplication = withRouter(injectIntl((ExampleApplication)));
 
 const AppRouter = () => (
-  <div className={cx('app-router')}>
-    <MemoryRouter>
-      <WrappedApplication />
-    </MemoryRouter>
+  <div className={cx('height-tag')}>
```

Comment:
Since these example's code can be toggled, can we add a descriptive class name to indicate this additional wrapper is only necessary for the example? Something like `height-needed-for-docs`? though probably a better name than that

Reply:
Changed